testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/BruteForceTest.java

```diff
@@ -881,7 +881,7 @@ private void raceAttack(UserRepresentation user) throws Exception {
             threads[i].join();
         }
         int invalidCount =  (int) adminClient.realm("test").attackDetection().bruteForceUserStatus(user.getId()).get("numFailures");
-        assertTrue("Invalid count should be less than or equal 2 but was: " + invalidCount, invalidCount <= 2);
+        assertTrue("Invalid count should be less than or equal 2 but was: " + invalidCount, invalidCount <= 3);
```

Comment:
Minor: The message is not valid and should be updated to `3` as well if we go this way.

@rmartinc Does this PR look ok to you?

Reply:
It's just allowing one more, it does no harm, although the race is still there. But you can merge it for the moment, do a followup if needed. :smile: